client/fns.go

```diff
@@ -200,6 +200,30 @@ func GetHostName(host string) string {
 	return host
 }
 
+// GetHostUser gets the user and host name. In ssh syntax, these can
+// be in one string, or defined in .ssh/config, or (for user), as
+// $USER. Given that just about anything can be valid, and errors
+// will be caught in other places, it does not return an error.
+// Also, experimentally, in the name@host form, name overrides
+// any use name that might be in .ssh/config. Host, on the other hand,
+// is always determined from .ssh/config. The host name
+// is also pulled from .ssh/config; calls to GetHostName
+// can be replaced as needed, but that's for the future.
+func GetHostUser(n string) (host, user string) {
+	if u, h, ok := strings.Cut(n, "@"); ok {
+		host, user = GetHostName(h), u
```

Comment:
You can return right away and make this function for 3 lines shorter.
```go
		return GetHostName(h), u
```
than 
```go
		host, user = GetHostName(h), u
 		return
```

Reply:
yep, thanks, was not thinking.